packages/server/api/model/supportReference.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const referenceSchema = new Schema({
+    link: {
+        type: String,
+        required: true
+    },
+    citation: {
+        type: String,
+        required: true
+    },
+    type: {
+        type: String,
+        required: true
+    },
+    classification: {
```

Comment:
Classification should an enum:

- Reliable source
- Unreliable source
- Fake